openrouteservice/client.py

```diff
@@ -225,9 +225,15 @@ def req(self):
     @staticmethod
     def _get_body(response):
         """Returns the body of a response object, raises status code exceptions if necessary."""
-        body = response.json()
-        # error = body.get('error')
         status_code = response.status_code
+        try:
+            body = response.json()
+        except json.JSONDecodeError:
+            if status_code != 200:
+                raise exceptions.ApiError(
```

Comment:
it's fine generally, thx. can you get rid of the redundant `status_code` and raise a `exceptions.HTTPError` instead pls? 

Background: you'll also get a `404` when a route couldn't be found, which actually is an ApiError. The `404` you're referring to should only arise when you request a false URL, like `https://api.openrouteservice.org/false`, which is likely if you set up a bad `base_url` in `Client`.